### ISSUE TITLE
HTTP headers support

### ISSUE THREAD

**Issue by thomkrupa** [bejamas/gatsby-plugin-csp#4]
https://github.com/gatsbyjs/gatsby/issues/10890#issuecomment-468982396

**Comment by ryanerringtonatom** [comment]
@lightningspirit I agree, this feature would make server-side CSP implementation a lot easier.

Do you know of any alternate solutions/workarounds for a header-based approach?

**Comment by lightningspirit** [comment]
> @lightningspirit I agree, this feature would make server-side CSP implementation a lot easier.
> 
> Do you know of any alternate solutions/workarounds for a header-based approach?

Actually, after a couple of tests, I ended up not using this plugin at all. Instead, I only used https://github.com/jariz/gatsby-plugin-s3/issues/144 and decided to hardcode my `Content-Security-Policy` in plugin's configs for headers.
